airflow/models/slamiss.py

```diff
@@ -44,3 +49,7 @@ class SlaMiss(Base):
 
     def __repr__(self):
         return str((self.dag_id, self.task_id, self.execution_date.isoformat()))
```

Comment:
Should the above self.execution_date be changed too?

Reply:
Doesn’t matter since the two point to the same thing. 2.10 has a lot of things using `execution_date`, let’s not bother changing this.